tests/utils.py

```diff
@@ -122,13 +122,17 @@ def test_case_3():
 class SchemaValidator:
     REPO = 'https://raw.githubusercontent.com/radiantearth/stac-spec'
     TAG = 'v{}'.format(STAC_VERSION)
-    SCHEMA_BASE_URI = '{}/{}'.format(REPO, TAG)
-
+    # SCHEMA_BASE_URI = '{}/{}'.format(REPO, TAG)
+    # Temporarily set to point to a fork if stac-spec with updated
+    # schemas for Label and EO
+    SCHEMA_BASE_URI = 'https://raw.githubusercontent.com/simonkassel/stac-spec/sk/refactor-extension-schemas'
+    
     schemas = {
         Catalog: 'catalog-spec/json-schema/catalog.json',
         Collection: 'collection-spec/json-schema/collection.json',
         Item: 'item-spec/json-schema/item.json',
-        LabelItem: 'extensions/label/schema.json',
+        LabelItem: 'extensions/label/json-schema/stac-label-item-schema.json',
+        EOItem: 'extensions/eo/json-schema/stac-extension-eo.json'
```

Comment:
More a comment on the stac-spec change, but I think this should try to match the other filenames, e.g.

```
json-schema/label-item.json
json-schema/eo-item.json
```

Since none of the others have  'stac' or 'schema' in them

Reply:
yeah, i think that makes sense, will change in both places